arex-storage-config/src/main/java/com/arextest/config/model/vo/UpdateApplicationRequest.java

```diff
@@ -15,4 +16,8 @@ public class UpdateApplicationRequest {
   private String appId;
   private String appName;
   private Set<String> owners;
+  /**
+   * @see com.arextest.model.replay.AppVisibilityLevelEnum .
+   */
+  private Integer visibilityLevel;
```

Comment:
the missing verification of the parameter

Reply:
Replace Integer with int to maintain consistency.